It's the Windows container initialization that's failing during CI.

@jonyMarino Just a heads up
```
2020-12-14T22:53:40.8901202Z 
##[section]Starting: Initialize containers
2020-12-14T22:53:41.0464982Z 
##[command]C:\Program Files\Docker\docker.EXE version --format '{{.Server.APIVersion}}'
2020-12-14T22:53:42.6313274Z error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.40/version: open //./pipe/docker_engine: The system cannot find the file specified. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.
2020-12-14T22:53:42.6315842Z '
2020-12-14T22:53:42.8171413Z 
##[error]Exit code 1 returned from process: file name 'C:\Program Files\Docker\docker.EXE', arguments 'version --format '{{.Server.APIVersion}}''.
2020-12-14T22:53:42.8190394Z 
##[section]Finishing: Initialize containers
```

--- a/PythonClient/airsim/client.py
+++ b/PythonClient/airsim/client.py
@@ -790,7 +790,7 @@ class VehicleClient:
         """
         self.client.call('simSetWind', wind)
 
-    def createVoxelGrid(self, position, x, y, z, res, of):
+    def simCreateVoxelGrid(self, position, x, y, z, res, of):
         """
         Construct and save a binvox-formatted voxel grid of environment
 

--- a/Unity/AirLibWrapper/AirsimWrapper/Source/WorldSimApi.cpp
+++ b/Unity/AirLibWrapper/AirsimWrapper/Source/WorldSimApi.cpp
@@ -103,6 +103,11 @@ void WorldSimApi::setWeatherParameter(WeatherParameter param, float val)
     //TODO: implement weather for Unity
 }
 
+bool WorldSimApi::createVoxelGrid(const Vector3r& position, const int& x_size, const int& y_size, const int& z_size, const float& res, const std::string& output_file)
+{
+    return false;
+}
+
 //----------------Plotting APIs-----------/
 void WorldSimApi::simFlushPersistentMarkers()
 {

--- a/Unity/AirLibWrapper/AirsimWrapper/Source/WorldSimApi.h
+++ b/Unity/AirLibWrapper/AirsimWrapper/Source/WorldSimApi.h
@@ -62,6 +62,7 @@ public:
     virtual bool isRecording() const override;
 
     virtual void setWind(const Vector3r& wind) const override;
+    virtual bool createVoxelGrid(const Vector3r& position, const int& x_size, const int& y_size, const int& z_size, const float& res, const std::string& output_file) override;
 
 private:
 	SimModeBase * simmode_;

